From 281935a1834df30fb68ffb3eea74d51492530bae Mon Sep 17 00:00:00 2001
From: Jason Liu <r64343@freescale.com>
Date: Mon, 19 Sep 2011 16:32:54 +0800
Subject: [PATCH 0430/2463] ENGR00157032 i.MX6/defconfig: enable ARM errata
 software workaround

We need enable the following ARM errata software workaround:

CONFIG_ARM_ERRATA_743622=y
CONFIG_ARM_ERRATA_751472=y

These ERRATAs applied to i.MX6Q(cortex-a9:r2p10 smp)

Signed-off-by: Jason Liu <r64343@freescale.com>
---
 arch/arm/configs/imx6_defconfig |    6 +++---
 1 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/arch/arm/configs/imx6_defconfig b/arch/arm/configs/imx6_defconfig
index 558d5bc..d40a01e 100644
--- a/arch/arm/configs/imx6_defconfig
+++ b/arch/arm/configs/imx6_defconfig
@@ -1,7 +1,7 @@
 #
 # Automatically generated make config: don't edit
 # Linux/arm 2.6.38 Kernel Configuration
-# Thu Sep 15 18:47:42 2011
+# Mon Sep 19 16:32:11 2011
 #
 CONFIG_ARM=y
 CONFIG_HAVE_PWM=y
@@ -353,8 +353,8 @@ CONFIG_CPU_HAS_PMU=y
 # CONFIG_ARM_ERRATA_742231 is not set
 # CONFIG_PL310_ERRATA_588369 is not set
 # CONFIG_ARM_ERRATA_720789 is not set
-# CONFIG_ARM_ERRATA_743622 is not set
-# CONFIG_ARM_ERRATA_751472 is not set
+CONFIG_ARM_ERRATA_743622=y
+CONFIG_ARM_ERRATA_751472=y
 # CONFIG_ARM_ERRATA_753970 is not set
 # CONFIG_PL310_ERRATA_727915 is not set
 CONFIG_ARM_GIC=y
-- 
1.7.7.4

